receiver/README.md

```diff
@@ -184,6 +188,54 @@ receivers:
     - https://*.example.com
 ```
 
+## <a name="otlp-traces"></a>OpenTelemetry Receiver
+
+This is the default receiver for the OpenTelemetry project.
+
+To get started, all that is required to enable the OpenTelemetry receiver is to
+include it in the receiver definitions. This will enable the default values as
+specified [here](otlpreceiver/factory.go).
+The following is an example:
+```yaml
+receivers:
+  otlp:
+```
+
+The full list of settings exposed for this receiver are documented [here](otlpreceiver/config.go)
+with detailed sample configurations [here](otlpreceiver/testdata/config.yaml).
+
+### Communicating over TLS
+This receiver supports communication using Transport Layer Security (TLS). TLS
+can be configured by specifying a `tls_credentials` object in the receiver
+configuration for receivers that support it.
+```yaml
+receivers:
+  otlp:
+    tls_credentials:
+      key_file: /key.pem # path to private key
+      cert_file: /cert.pem # path to certificate
+```
+
+### Writing with HTTP/JSON
```

Comment:
Add a comment that the JSON needs to be the protobuf JSON serialization from https://developers.google.com/protocol-buffers/docs/proto3#json

Maybe add a short `IMPORTANT: bytes fields are encoded as base64 strings`